modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsSnapshot.java

```diff
@@ -398,7 +398,7 @@ public ClusterMetricsSnapshot(ClusterGroup p) {
 
             nonHeapInit += m.getNonHeapMemoryInitialized();
 
-            upTime = max(upTime, m.getUpTime());
+            upTime += m.getUpTime();
```

Comment:
What about `max(0, m.getUpTime())` ?

Reply:
The logic of cluster metrics is considered as the sum of the nodes, and in this case it was the maximum among the nodes, which breaks all further calculation. actually, this is the correction for the ticket, everything else is added to protect and edit inconsistencies in the name and content.

Reply:
It seems that m.getUpTime() can be `-1` in some cases (if not initialized) but if you sure it can't - let's leave it as is.

Reply:
UpTime solve in ClusterMetricsImpl 
MetricRegistry mreg = ctx.metric().registry(SYS_METRICS);
upTime = mreg.findMetric(UP_TIME);
... 
in GridMetricManager
sysreg.register(UP_TIME, rt::getUptime, null);
...
java.lang.management.RuntimeMXBean#getUptime
sun.management.VMManagement#getUptime
...
Finally
Returns the uptime of the Java virtual machine in milliseconds.